src/core/client/ui/components/v2/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,39 @@
+import cn from "classnames";
+import { Icon } from "coral-ui/components";
+import React, { AnchorHTMLAttributes, StatelessComponent } from "react";
+
+import { withStyles } from "coral-ui/hocs";
+
+import styles from "./TextLink.css";
+
+interface Props extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes: typeof styles;
+}
+
+const TextLinkProps: StatelessComponent<Props> = props => {
+  const { className, children, classes, target, ...rest } = props;
+
+  const rootClassName = cn(classes.root, className);
+
+  return (
+    <a
+      className={rootClassName}
+      href={props.href || (children as string)}
+      target={target}
```

Comment:
Anchors that contain a target should also contain a `rel=` prop.

Reply:
this is just the old textlink component moved to a new location, but it looks like its just using `AnchorHTMLAttributes` for props, `target` is pulled out from `...rest` so it can be used conditionally, but I can let it get passed through with `...rest` and just reference `props.target`